subsys/net/l2/Kconfig

```diff
@@ -76,6 +76,11 @@ source "subsys/net/l2/openthread/Kconfig"
 
 source "subsys/net/l2/canbus/Kconfig"
 
+config NET_L2_ZIGBEE
+	bool "Zigbee L2"
+	depends on NETWORKING
+	imply NET_PKT_TIMESTAMP
```

Comment:
since zigbee needs timestamps this should be select or depends on

Reply:
changed to `SELECT`